go.mod

```diff
@@ -113,6 +113,7 @@ require (
 	github.com/jezek/xgb v0.0.0-20210312150743-0e0f116e1240 // indirect
 	github.com/jinzhu/copier v0.3.4 // indirect
 	github.com/joho/godotenv v1.5.1 // indirect
+	github.com/kairos-io/kairos v2.3.0+incompatible // indirect
```

Comment:
and it also now becomes "incompatible"

Reply:
+incompatible means the dependency has a semver major version of 2 or higher and is not a Go module yet (it doesn't have go.mod in its source code).

But this doent seem to match the status...Maybe it does because under kairos repo there is no longer a go.mod file...

Reply:
yeah I was wondering, but I think the change has been there for a few versions now, so I'm not sure why it just now becomes incompatible. Not really a fan of go versions "yet" haha

Reply:
Im wondering something else depends on that and it just brings it like that? probabyl due to a replace directive in the other dep go.mod?

go deps are worst than python deps, omg.